apis/python/src/tiledbsoma/io/shaping.py

```diff
@@ -46,14 +51,11 @@ def show_experiment_shapes(
     context: tiledbsoma.SOMATileDBContext | None = None,
     output_handle: Printable = cast(Printable, sys.stdout),
 ) -> bool:
-    """For each dataframe/array contained within the SOMA ``Experiment`` pointed
-    to by the given URI, shows the ``non_empty_domain`` (for dataframes), along
-    with the ``shape`` and ``maxshape`` (for arrays) or ``domain`` and
-    ``maxdomain`` (for dataframes).
+    """Outputs the current shapes of the main elements in the ``Experiment``.
 
-    Args:
-        uri: The URI of a SOMA :class:`Experiment`.
-        context: Optional :class:`SOMATileDBContext`.
+    For main dataframes and arrays contained within the SOMA ``Experiment``, shows
```

Comment:
```suggestion
    For dataframes and arrays contained within the SOMA ``Experiment``, shows
```

I'm not sure what 'main' means (what are the things that are not main?) -- so I suggest we omit that word.